scripts/buildDemo.js

```diff
@@ -127,15 +127,15 @@ const getCategory = (iconPath) => {
   <meta charset="utf-8">
 
   <title>Icons demo page</title>
-  <link rel="stylesheet" href="https://unpkg.com/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
 
   <style>
     .block {
-      font-size: 12px;
+      font-size: var(--ds-size-150);
```

Comment:
Switching from a fixed size to a responsive size may be an issue. Are we sure this isn't a problem?

Reply:
This is only for an internally created demo page. There is no risk of consumer issues.